docs-source/cloudbank/content/provision/install-free-trial.md

```diff
@@ -148,3 +148,72 @@ You can also use this approach in any regular commercial Oracle Cloud Tenancy -
    * The Spring Operations Center main dashboard will appear similar to this:
 
      ![Spring Operations Center](../images/install-free-tier-5.png " ")
+
+1. DB Access & setup
+
+   * Connect to your instance (using SSH), find the database pod with the following command: `kubectl get pods --all-namespaces`.
+
+      In the `oracle-database-operator-systemthen the address` namespace, look for a pods like `free-XXXX`.
```

Comment:
This doesn't look right to me? `oracle-database-operator-systemthen the address`

Reply:
fixed